utils/src/main/kotlin/Utils.kt

```diff
@@ -134,9 +134,10 @@ fun filterVersionNames(version: String, names: List<String>): List<String> {
                 name.endsWith(versionName) -> {
                     val head = name.removeSuffix(versionName)
                     val last = head.lastOrNull()
-                    val nextToLast = head.dropLast(1).lastOrNull()
-                    last == null ||
-                            (last in versionElementSeparators && (nextToLast == null || !nextToLast.isDigit()))
+                    val forelast = head.dropLast(1).lastOrNull()
```

Comment:
Better name it "penultimate", "forelast" is not in any dictionary I checked.

Reply:
I was using `nextToLast` before, which I slightly prefered, but then the line became too long and I did not want to wrap it 😆 And you need to use the right dictonary 😉 https://www.dict.cc/?s=forelast